config/client.lua

```diff
@@ -1,6 +1,8 @@
 return {
     useTarget = false,
 
+    fingerprintChance = 50, -- Chance of dropping a fingerprint if not wearing gloves
```

Comment:
Optional; Would this be better as 0.5 then 50? As in, is it better to express percentages as decimals?

Reply:
I have no idea

Reply:
If it was 0.5 then the math.random needs to be changed to just () vice specifying the int values. From a _config dev_ point of view the 50 is clearer to understand.